### PR TITLE
Enable manually triggering the image build

### PR DESCRIPTION
**Pull request by csomh**
... and set up a bi-weekly scheduled trigger.

### CODE REVIEW

**Review by mfocko — APPROVED** [review]
Nice, catches February too 😯

File: .github/workflows/build-and-push.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name: Rebuild & push images
 on:
+  workflow_dispatch:
```
Comment (jpopelka) [inline .github/workflows/build-and-push.yml:3]: nice, can you add it also to [the template](https://github.com/packit/.github/blob/main/workflow-templates/build-and-push.yml)?

**Review by jpopelka — APPROVED** [review]

**Comment by csomh** [comment]
recheck

File: .github/workflows/build-and-push.yml (hunk 1 above)
Comment (csomh) [reply .github/workflows/build-and-push.yml:3]: :+1:  packit/.github#4

**Comment by csomh** [comment]
regate

**Pull request merged**